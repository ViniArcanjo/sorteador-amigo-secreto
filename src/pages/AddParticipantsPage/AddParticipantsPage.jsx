import Card from '../../components/Card';
import { Form } from '../../components/Form/Form';

export function AddParticipantsPage() {
  return (
    <Card>
      <Form />
    </Card>
  );
}
