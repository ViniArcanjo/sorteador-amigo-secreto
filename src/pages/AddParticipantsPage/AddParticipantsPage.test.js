import { render } from '@testing-library/react';

import FormProvider from '../../context/FormContext/FormContext';
import { AddParticipantsPage } from './AddParticipantsPage';

const mockUseNavigate = jest.fn();

jest.mock('react-router-dom', () => {
  return {
    useNavigate: () => mockUseNavigate,
  };
});

describe('<StartConfigPage />', () => {
  it('deve renderizar todos os componentes na ordem correta', () => {
    const { container } = render(
      <FormProvider>
        <AddParticipantsPage />
      </FormProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
