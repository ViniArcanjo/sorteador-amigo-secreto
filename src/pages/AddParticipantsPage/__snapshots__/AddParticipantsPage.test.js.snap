// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StartConfigPage /> deve renderizar todos os componentes na ordem correta 1`] = `
<div>
  <div
    class="card_wrapper"
  >
    <div
      class="content_wrapper"
    >
      <form
        class="form_wrapper"
      >
        <h1>
          Vamos Come√ßar!
        </h1>
        <div>
          <img
            src="person_add.png"
          />
          <input
            placeholder="Insira o nome dos participantes"
            type="text"
            value=""
          />
        </div>
        <button
          disabled=""
        >
          Adicionar
        </button>
      </form>
      <ul
        class="list_wrapper"
      />
      <button
        disabled=""
      >
        Iniciar Brincadeira!
      </button>
      <img
        src="sacolas.png"
      />
    </div>
  </div>
</div>
`;
